homeassistant/components/miflora/sensor.py

```diff
@@ -158,9 +158,11 @@ def update(self):
             data = self.poller.parameter_value(self.parameter)
         except OSError as ioerr:
             _LOGGER.info("Polling error %s", ioerr)
+            self._state = None
```

Comment:
Instead of `self._state = None`, we should return `False` from the `available` property.